requirements.txt

```diff
@@ -1,8 +1,8 @@
 click >= 7.0, < 8.0
 cloudpickle >=0.6.0, <2.0
 croniter >= 0.3.24, <1.0
-dask[bag] >=0.19.3, <3.0
-distributed >= 1.26.1, < 3.0
+dask[bag] >= 2.8.0, <3.0
```

Comment:
I actually don't think we use `dask[bag]` anymore so we should be able to remove `[bag]` here